lms/djangoapps/discussion_api/tests/test_views.py

```diff
@@ -519,6 +519,86 @@ def test_order_direction(self, query):
             "sort_order": [query],
         })
 
+    def test_discussion_id(self):
```

Comment:
I think this test is duplicate of `test_basic` except few values, can we merge this into one ?